prime-angular-frontend/src/app/modules/adjudication/shared/components/ha-authorized-users-view/ha-authorized-users-view.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { MatDialog } from '@angular/material/dialog';
+import { EMPTY, noop, of, Subscription } from 'rxjs';
+import { exhaustMap } from 'rxjs/operators';
+
+import { RouteUtils } from '@lib/utils/route-utils.class';
+
+import { HealthAuthorityResourceService } from '@core/resources/health-authority-resource.service';
+import { HAAuthorizedUser } from '@shared/models/ha-authorized-user.model';
+import { FormatDatePipe } from '@shared/pipes/format-date.pipe';
+import { DialogOptions } from '@shared/components/dialogs/dialog-options.model';
+import { ConfirmDialogComponent } from '@shared/components/dialogs/confirm-dialog/confirm-dialog.component';
+
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+
+@Component({
+  selector: 'app-ha-authorized-users-view',
+  templateUrl: './ha-authorized-users-view.component.html',
+  styleUrls: ['./ha-authorized-users-view.component.scss']
+})
+export class HaAuthorizedUsersViewComponent implements OnInit {
+  public busy: Subscription;
+  public authorizedUsers: HAAuthorizedUser[];
+
+  private routeUtils: RouteUtils;
+
+  constructor(
+    protected route: ActivatedRoute,
```

Comment:
Should be `private` only need to use `protected` for DI when passing to a base class through `super(...)`